checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsAnnotatedTypeFactory.java

```diff
@@ -112,7 +112,7 @@ public CalledMethodsAnnotatedTypeFactory(BaseTypeChecker checker) {
    *
    * @param disabledFrameworks the disabled builder frameworks
    */
-  private void enableFrameworks(String[] disabledFrameworks) {
+  private final void enableFrameworks(String[] disabledFrameworks) {
```

Comment:
`final` on a `private` method is redundant.  Is there a reason you like to add it?